.circleci/config.yml

```diff
@@ -1710,8 +1752,8 @@ workflows:
               op_component:
                 - op-node
                 - op-batcher
+                - op-program
                 - op-proposer
-                - op-challenger
```

Comment:
Missing context question - why are we dropping op-challenger here?

Reply:
oops, bad diff I think. I meant to drop op-acceptance-tests. Will fix.

Reply:
Fixed in f84dc9a8